### DISCUSSION TITLE
Error when generating the Avalon template build

### DISCUSSION THREAD

**Discussion by MarcelaTMS** [primefaces/community#1118] [PrimeReact Templates] [answered]
How do I generate the build of the Avalon template of primereact with next.js. I already have sass installed as the documentation indicates and I get this error
./styles/demo/Demos.scss
To use Next.js' built-in Sass support, you first need to install `sass`.
Run `npm i sass` or `yarn add sass` inside your workspace.

Learn more: https://nextjs.org/docs/messages/install-sass

./styles/layout/layout.scss
To use Next.js' built-in Sass support, you first need to install `sass`.
Run `npm i sass` or `yarn add sass` inside your workspace.

**Comment by buraksaglam089** [comment, marked as answer]
Hello,

Firstly, ensure that the files are imported correctly. Then, compile the Sass files using the Sass command to generate the CSS files. If you encounter the same error after completing these steps, please inform me.

Regards
